packages/elements/src/styles/elements.scss

```diff
@@ -13,6 +13,10 @@ html {
   font-size: 13px;
 }
 
+a:hover {
```

Comment:
Is there a better way to solve this problem?

I believe at some point we hoped to "unscope" elements styles, and this would become an issue in that case.

Reply:
This issue is coming from the default styles in UI-Kit, which we're working towards removing. This is a temporary fix until we can remove UI-Kit completely over the next few sprints.